ui/app/components/global-search/control.js

```diff
@@ -62,15 +66,17 @@ export default class GlobalSearchControl extends Component {
       set(this, 'nodes', nodes.toArray());
 
       const jobResults = this.jobSearch.listSearched.slice(0, MAXIMUM_RESULTS);
-      const nodeResults = this.nodeSearch.listSearched.slice(0, MAXIMUM_RESULTS);
+
+      const mergedNodeListSearched = this.nodeNameSearch.listSearched.concat(this.nodeIdSearch.listSearched).uniq();
```

Comment:
I think if someone is searching by ID and it happens to also match a job, it should get priority rather than being concatenated after the name matches.

Reply:
Do you mean happens to also match a node name? If so, agreed, I can make that change and add a test for it, thanks.

Reply:
Ha, yes, node name. Too much multitasking.

Reply:
okay, done in fc47bb8 😀